int32 r_height
int32 l_height
int32 t_length
int32 b_length
float c_pos
string qr_id
